VRDR.Messaging/AliasMessage.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Hl7.Fhir.Model;
+
+namespace VRDR
+{
+    /// <summary>Class <c>AliasMessage</c> indicates that Alias information is being provided for a previously submitted DeathRecordSubmission message.
+    ///    Modeled on void message.
+    /// </summary>
+    public class AliasMessage : BaseMessage
```

Comment:
are all the alias fields part of the generated code or base message and that's why there's no fields in this class?